react-client-app/src/components/Home.jsx

```diff
@@ -184,6 +184,35 @@ const Home = () => {
         return JSON.parse(window.status).find((item) => item.id == status).label;
     };
 
+    const getGraphData = () => {
+        const stat = JSON.parse(window.status)
+        const data = []
+        const dataPartners = [...new Set(scanreportsRef.current.map(data => data.data_partner.name))].sort((a, b) => a.localeCompare(b))
+        dataPartners.map(item => {
+            data.push(
+                {
+                    x: stat.map((status) => status.label),
+                    y: stat.map((status) => statuses.find(sta => status.id == sta.id)
+                        .data.filter(report => report.data_partner.name == item).length),
+                    type: "bar",
+                    name: item,
+                    xaxis: 'x1',
+                    barmode: 'stack',
+                    //marker: { color: stringToColour(item) } // uncomment this to match colours with other graphs
```

Comment:
Please uncomment this as you say to match the other colours, then we're good to go

Reply:
Done